### PR TITLE
167951782 reconfigure office queues

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Description

Reconfigure the queues in the office app to reflect requested changes. More details in the story.

## Setup

No special steps beyond the usual e2e population.

```sh
make db_dev_e2e_populate
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/167951782) for this change

## Screenshots

image

image

image

image

image

### CODE REVIEW

File: cypress/integration/office/officeUserPPM.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ describe('office user finds the move', function() {
 
   it('edits pickup and destination zip codes in estimates panel and these values are reflected in the storage and incentive calculators', function() {
     officeUserGoesToPPMPanel('FDXTIU');
-    officeUserEditsEstimatesPanel(60606, 72018, 6000);
+    officeUserEditsEstimatesPanel(60606, 72018, 5000);
```
Comment (lynzt) [inline cypress/integration/office/officeUserPPM.js]: just curious why the change from `6000` to `5000`

File: src/scenes/Office/QueueList.jsx [hunk 2]
```diff
@@ -3,49 +3,38 @@ import { NavLink } from 'react-router-dom';
...
-                  <span>Payment requests</span>
-                </NavLink>
-              </li>
-            </ul>
           </li>
           <li>
-            <NavLink to="/queues/all" activeClassName="usa-current">
-              <span>All Moves</span>
+            <NavLink to="/queues/all" activeClassName="usa-current" data-cy="ppm-queue">
+              <span>All</span>
```
Comment (lynzt) [inline src/scenes/Office/QueueList.jsx]: double-check w/ design that we want to have `All` vs `All Moves`

File: src/scenes/Office/QueueList.jsx [hunk 3]
```diff
@@ -3,49 +3,38 @@ import { NavLink } from 'react-router-dom';
...
         <ul className="usa-sidenav-list">
           <li>
             <NavLink to="/queues/new" activeClassName="usa-current">
-              <span>New Moves/Shipments</span>
+              <span>New</span>
+            </NavLink>
+          </li>
+
+          <li>
+            <NavLink to="/queues/ppm_approved" activeClassName="usa-current" data-cy="ppm-payment-requests-queue">
```
Comment (garrettqmartin8) [inline src/scenes/Office/QueueList.jsx]: Yay for `data-cy`

**Comment by garrettqmartin8** [comment]
Good stuff - once the things @lynzt mentioned are addressed, this should be good to go.

File: src/scenes/Office/QueueList.jsx (hunk 2 above)
Comment (Ryan-Koch) [reply src/scenes/Office/QueueList.jsx]: I'll revert that back for now.

File: cypress/integration/office/officeUserPPM.js (hunk 1 above)
Comment (Ryan-Koch) [reply cypress/integration/office/officeUserPPM.js]: Ah I think this is an artifact of my troubleshooting something. Changed it back.

**Comment by Ryan-Koch** [comment]
> nice work...
> It would be great if you could take a look at `move_queue_items_test.go` to make sure we're testing what we think we are testing (and not having false positives).
> This isn't something in the story, but I would love to have these queries better tested and this pr seems like a good place for that.
> `TestShowPPMPaymentRequestsQueue` is testing the status of the returned move which is something I'd like to do in these tests vs just testing a count of the number of recs we get back...

I added something to `pkg/handlers/internalapi/move_queue_items_test.go` to have it take the currently available statuses in the test and then check the status during each loop (going in versus what's coming out). Does this achieve what you were hoping?
